docs-src/0.6/src/new-guide/backend.md

```diff
@@ -0,0 +1,90 @@
+# Adding a Backend
+
+Dioxus is a *fullstack* framework, meaning it allows you to seamlessly build your frontend alongside your backend.
+
+We provide a number of utilities like *Server Functions*, *Server Futures*, and *Server State* for you to integrate into your apps. In this chapter, we'll cover loading and saving state to our backend with *Server Functions*. For an in-depth guide on fullstack, check out the dedicated [Fullstack Guide](../guides/fullstack/index.md).
+
+## Enabling Fullstack
+
+Before we can start using server functions, we need to enable the "fullstack" feature on Dioxus in our Cargo.toml.
+
+```toml
+[dependencies]
+dioxus = { version = "0.6.0", features = ["fullstack"] }
+```
+
+We also need to add the "server" feature to our app's features, also in the Cargo.toml
+
+```toml
+[features]
+default = ["web"]
```

Comment:
We need to remove the web feature from the default features list when switching to fullstack so it isn't enabled on the server